tests/integration/annotation_import/test_export_v2_classifications.py

```diff
@@ -0,0 +1,181 @@
+import labelbox as lb
```

Comment:
this is a more thorough test but I have made a note to merge it with the main `test_import_data_types_v2` during my next story to test DICOM